### ISSUE TITLE
Javascript doenot work for other data but first page

### ISSUE THREAD

**Issue by kirtesh08** [pklauzinski/jscroll#71]
While using jscroll I have added some other javascript to my objects which seems not working for any of the next set of data but working for only first set

**Comment by hakimihamdan88** [comment]
yeah, any idea? im having problem on 2nd page(ajax loaded) with timeago.js too
